pulsar/resource_pulsar_function.go

```diff
@@ -327,19 +351,19 @@ func resourcePulsarFunction() *schema.Resource {
 			resourceFunctionCPUKey: {
 				Type:        schema.TypeFloat,
 				Optional:    true,
-				Default:     0.5,
+				Computed:    true,
```

Comment:
why the default value is removed?

Reply:
the resources will be returned by worker service apis and it may changed by the worker service, so it should be marked as `computed` to prevent the resources been frequently update.